Source/ConfigCat/Private/ConfigCatSubsystem.cpp

```diff
@@ -217,20 +211,20 @@ void UConfigCatSubsystem::ForceRefresh()
 	ConfigCatClient->forceRefresh();
 }
 
-void UConfigCatSubsystem::SetDefaultUser(const FConfigCatUser& User)
+void UConfigCatSubsystem::SetDefaultUser(const UConfigCatUserWrapper* User)
 {
 	if (!ensure(ConfigCatClient))
```

Comment:
Could you please create an EnsureConfigCatClient() function where this log message can be printed:
```
bool EnsureConfigCatCline()
{
	if (ensure(ConfigCatClient))
	{
                return true;
        }
        
	UE_LOG(LogConfigCat, Warning, TEXT("Trying to access the ConfigCatClient before initialization or after shutdown."));
	return false;
}
```

In every function we can use this EnsureConfigCatClient() to check the client easier:

```
if (!EnsureConfigCatClient())
{
        return;
}
```

Reply:
That is a great idea. I added it now, thanks.